client/src/main/java/io/split/client/impressions/UniqueKeysTrackerImp.java

```diff
@@ -0,0 +1,64 @@
+package io.split.client.impressions;
+
+import io.split.client.impressions.filters.BloomFilterImp;
+import io.split.client.impressions.filters.Filter;
+import io.split.client.impressions.filters.FilterAdapter;
+import io.split.client.impressions.filters.FilterAdapterImpl;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.HashSet;
+import java.util.concurrent.ConcurrentHashMap;
+
+public class UniqueKeysTrackerImp implements UniqueKeysTracker{
+    private static final double MARGIN_ERROR = 0.01;
+    private static final int MAX_AMOUNT_OF_TRACKED_MTKS = 30000;
+    private static final int MAX_AMOUNT_OF_KEYS = 10000000;
+    private FilterAdapter filterAdapter;
+    private ImpressionsSender impressionsSender;
+    private final ConcurrentHashMap<String,HashSet<String>> mtkTracker;
+    private static final Logger _logger = LoggerFactory.getLogger(HttpImpressionsSender.class);
```

Comment:
the name should be `UniqueKeysTrackerImp.class` instead of `HttpImpressionSender.class`

Reply:
I changed it.